pkg/runtime/types.go

```diff
@@ -96,10 +100,16 @@ type RawExtension struct {
 // +protobuf=true
 type Unknown struct {
 	TypeMeta `json:",inline"`
-	// RawJSON will hold the complete JSON of the object which couldn't be matched
+	// Raw will hold the complete serialized object which couldn't be matched
 	// with a registered type. Most likely, nothing should be done with this
 	// except for passing it through the system.
-	RawJSON []byte
+	Raw []byte
```

Comment:
@smarterclayton - I guess we should also change RawExtension. Can I do it in a separate PR?